src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v7/ConfigV7.java

```diff
@@ -132,8 +134,8 @@ public String toString() {
         }
     }
 
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
Please add unit tests for PATCH around this

Reply:
Adding integration tests for PATCH is outside of scope for the PR, I'll add unit test for `ConfigV7.Kibana` serialization.

Reply:
Okay can you add tests for serialization and deserialization then?

Reply:
I added unit tests for `ConfigV6.Kibana` and `ConfigV7.Kibana`.